osf/migrations/0038_ensure_schemas.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-05-24 19:33
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations
+from osf.utils.migrations import ensure_schemas, remove_schemas
+
+
+logger = logging.getLogger(__file__)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0037_ensure_licenses'),
+    ]
+
+    operations = [
+        migrations.RunPython(ensure_schemas, remove_schemas),
```

Comment:
In the future, we can probably put ensure_schemas and ensure_licenses in a single migration, but this is fine for now